releasenotes/notes/0.19/bip-improvement-34cd623294a30d83.yaml

```
@@ -1,13 +1,13 @@
 ---
 features:
   - |
-    The ``balanced`` objective function of the ``BIPMapping`` no longer
-    requires a ``BackendProperties``. It will use ``BackendProperties`` if
+    The ``balanced`` objective function of the :obj:`.BIPMapping` no longer
+    requires a :obj:`.BackendProperties`. It will use :obj:`.BackendProperties` if
     available, otherwise it assumes a CX error rate as given in the new
     parameter ``default_cx_error_rate``. A new parameter ``depth_obj_weight``
     now controls the weight of the two objectives combined in the ``balanced``
     objective.
 upgrade:
   - |
-    The default method for ``BIPMapping`` is now ``balanced`` rather than
+    The default method for :obj:`.BIPMapping` is now ``balanced`` rather than
     ``depth``.
```

I would expand on this a little bit to make it explicit what to do and why this was done.

```suggestion
    The default method for :obj:`.BIPMapping` is now ``balanced`` rather than
    ``depth``. This was done as balanced achieves a generally better result as it
    factors in more than just circuit depth as the objective. If you require the previous
    behavior you should explicitly set the ``method`` kwarg on the :obj:`.BIPMapping`
    to ``"depth"``
```